cronjobs/cnn_classify_data.py

```diff
@@ -349,13 +375,65 @@ def match_files(cafiles, zbfiles):
                         outfits = '{}/transient_candidate_spectra_{}_{}.fits'.format(out_path, obsdate, tile_number)
                         write_spectra(outfits, cand_spectra)
                         print('Output file saved in {}'.format(outfits))
+                        
+                        
+#                         print(allflux)
+#                         print(fmap)
+
+                        #DESITRIP_daily - Send the selected fluxes to SkyPortal - Divij Sharma
+                        for i in range(len(fmap['TARGETID'])):
+                            SkyPortal.postCandidate(i, fmap)
+                            SkyPortal.postSpectra(fmap['TARGETID'][i].astype('str'), cand_spectra)
+                            
+        
+                            
+#                             logic.plotter(idx, _flux, savepdf=spdf)
 
-                        # Make a plot of up to 16 transients
 
-                        selection = sorted(np.random.choice(idx.flatten(), size=np.minimum(len(idx), 16), replace=False))
+                        # Make a plot of up to 16 transients
 
-                        fig, axes = plt.subplots(4,4, figsize=(15,10), sharex=True, sharey=True,
-                                                 gridspec_kw={'wspace':0, 'hspace':0})
+#                         selection = sorted(np.random.choice(idx.flatten(), size=np.minimum(len(idx), 16), replace=False))
+
+#                         fig, axes = plt.subplots(4,4, figsize=(15,10), sharex=True, sharey=True,
+#                                                  gridspec_kw={'wspace':0, 'hspace':0})
+
+
+#                         for j, ax in zip(selection, axes.flatten()):
+#                             ax.plot(rewave, rsflux[j], alpha=0.7, label='label: '+label_names[labels[j]]+'\nz={:.2f}'.format(allzbest[j]['Z']))
+#                             ax.legend(fontsize=10)
+
+#                         fig.tight_layout()
+#                         outplot = '{}/transient_candidates_{}_{}.png'.format(plot_path, obsdate, tile_number)
+#                         fig.savefig(outplot, dpi=200)
+#                         print('Figure saved in {}', outplot)
+                        
+#                     #Now add this tile info to the sqlite db
+#                     #Maybe expid is important for spectraldiff? Here we coadd
+#                     #expids = set([int(cframefile.split('-')[-1][:-5]) for cframefile in cframefiles])
+#                     prog=cframe_fibermap['PROGRAM']
+#                     sql = """ INSERT OR IGNORE INTO desitrip_exposures(tileid,program,obsdate)
+#                   VALUES(?,?,?) """
+#                     c.execute(sql,(tile_number, prog, obsdate))
+
+#             else:
+#                 print('Not a BGS tile')
```

Comment:
I am confused as to which lines I am supposed to uncomment to recover the previous version and have the db and plots updated?

Reply:
All of these lines existed in the original code, and I commented them out. So I would think uncommenting all of them.

Reply:
The pull version should be the version that works, not a test version.